.github/dependabot.yml

```diff
@@ -26,3 +26,45 @@ updates:
       prefix: ":seedling:"
     labels:
       - "ok-to-test"
+  - package-ecosystem: "gomod"
+    directory: "/api"
```

Comment:
would love to see if directory could take an array of paths to folders, but that seems to be a limitation https://github.com/dependabot/dependabot-core/issues/2178 at least for now

Reply:
Yes. It’s a pity. Already  feeling annoyed since this would most probably open separate PRs

Reply:
Let's see what sort of PR storm we're about to be hit with :)